grails-app/conf/Config.groovy

```diff
@@ -218,6 +214,40 @@ knownServers = [
         wmsVersion: '1.3.0',
         type: 'ncWMS'
     ],
+    [
+        uri: 'https://biocache-lb.ala.org.au/ws/ogc/getFeatureInfo',
+        type: 'ALAGetFeatureInfo'
+    ],
+    [
+        uri: 'https://biocache-lb.ala.org.au/ws/occurrences/offline/download',
+        type: 'ALADownload'
+    ],
+    [
+        uri: 'http://biocache-lb.ala.org.au/ws/ogc/wms/reflect',
+        wmsVersion: '1.1.1',
+        type: 'ALA'
+    ],
+    [
+        uri: 'https://biocache-lb.ala.org.au/ws/autocomplete/search',
```

Comment:
Do wee need all these entries?   Should just be able to allow requests to https://biocache-lb.ala.org.au/ws